core/runtime.rs

```diff
@@ -1858,6 +1849,19 @@ impl JsRuntime {
       .map(|handle| v8::Local::new(tc_scope, handle))
       .expect("ModuleInfo not found");
     let mut status = module.get_status();
+    if status == v8::ModuleStatus::Evaluated {
+      let (sender, receiver) = oneshot::channel();
+      sender.send(Ok(())).unwrap();
+      return receiver;
+    }
+    if status == v8::ModuleStatus::Errored {
+      let (sender, receiver) = oneshot::channel();
+      let exception = module.get_exception();
+      sender
+        .send(exception_to_err_result(tc_scope, exception, false))
+        .unwrap();
+      return receiver;
+    }
```

Comment:
I really don't like this bit - while it might be fine for our own internal code, this will hide problems in module loading logic for user code